README.md

```diff
@@ -3,6 +3,10 @@
   The GDX Analytics Drupal Snowplow module installs and runs Snowplow 
```

Comment:
needs to change to 'This is the central repository for work by the GDX Analytics Team.'

Reply:
Done.